packages/core-ui/lib/core/index.tsx

```diff
@@ -226,3 +227,29 @@ export const ButtonRow = ({ children, style, yMargin }: RowProps) => (
     {children}
   </div>
 );
+
+/* RowPrimaryOrRowSecondary
+ *
+ * The primary/secondary module in a PageRow.
+ */
+export const RowPrimaryOrRowSecondary = ({
+  children,
+  style,
+  verticalOrder,
+  paddingLeft,
+  secondary,
+}: RowPrimaryOrRowSecondaryProps) => (
+  <div
+    className={`${valOr(
+      secondary,
+      "core-row-secondary",
+      "core-row-primary"
```

Comment:
This should either by primary or secondary, not both. Again, need to be 2 separate components.